static/app/components/autoSelectText.tsx

```diff
@@ -25,45 +25,40 @@ type Props = {
   style?: CSSProperties;
 };
 
-class AutoSelectText extends React.Component<Props> {
-  private el: HTMLElement | undefined;
+function AutoSelectText({children, className, ...props}: Props) {
+  const element = useRef<HTMLElement>();
```

Comment:
Should we be more specific and be `HTMLSpanElement`?

Reply:
```suggestion
  const element = useRef<HTMLSpanElement>(null);
```